packages/services/src/view-service/fees.ts

```diff
@@ -60,9 +86,46 @@ export const extractAltFee = async (
   if (auctionWithdrawAsset) {
     const withdrawAuction = await indexedDb.getAuction(auctionWithdrawAsset);
     if (withdrawAuction.auction?.input?.assetId) {
-      return withdrawAuction.auction.input.assetId;
+      const checkAssetBalance = await indexedDb.hasTokenBalance(
+        request.source,
+        withdrawAuction.auction.input.assetId,
+      );
+      if (checkAssetBalance) {
+        return withdrawAuction.auction.input.assetId;
+      } else {
+        const assetId = await getAssetFromGasPriceTable(request, indexedDb);
+        if (assetId) {
+          return assetId;
+        }
+      }
+    }
+  }
+
+  if (request.undelegations.length > 0) {
+    const assetId = await getAssetFromGasPriceTable(request, indexedDb);
+    if (assetId) {
+      return assetId;
     }
   }
 
   throw new Error('Could not extract alternative fee assetId from TransactionPlannerRequest');
```

Comment:
Can we give alternative error, something user-readable like:
```
Not able to pay fee for transaction
```

Reply:
changed